packages/perseus/src/widgets/sorter/score-sorter.test.ts

```diff
@@ -29,16 +29,4 @@ describe("scoreSorter", () => {
         const result = scoreSorter(userInput, rubric);
         expect(result).toHaveBeenAnsweredIncorrectly();
     });
-
-    it("is invalid when the user has not made any changes", () => {
-        const userInput: PerseusSorterUserInput = {
-            options: ["$15$ grams", "$55$ grams", "$0.005$ kilograms"],
-            changed: false,
-        };
-        const rubric: PerseusSorterRubric = {
-            correct: ["$0.005$ kilograms", "$15$ grams", "$55$ grams"],
-        };
-        const result = scoreSorter(userInput, rubric);
-        expect(result).toHaveInvalidInput();
-    });
```

Comment:
I think we could keep a variant of this test in each scorer's test suite.  But, we can use mocking to ensure that `validate` is called first... something like:

I think we could keep a variant of this test in each scorer's test suite.  But, we can use mocking to ensure that `validate` is called first... something like:

```ts
// Top of file 
import * as SorterValidator from './validate-sorter'

    // Inside describe() block...
    it("should score if validator passes", () => {
        // Mock validator saying "all good" 👍 
        const mockValidate = jest
            .spyOn(SorterValidator, "default")
            .mockReturnValue(null);


        const score = scoreSorter(userInput, rubric);

        // Assert
        expect(mockValidate).toHaveBeenCalled();
        expect(score).toHaveBeenAnsweredCorrectly();
    });

    it("should abort if validator returns invalid", () => {
        // Mock validator saying "all good" 👍 
        const mockValidate = jest
            .spyOn(SorterValidator, "default")
            .mockReturnValue({type: "invalid", message: null});

        const score = scoreSorter(userInput, rubric);

        // Assert
        expect(mockValidate).toHaveBeenCalled();
        expect(score).toHaveInvalidInput()
    });
```